source/base/parameter_handler.cc

```diff
@@ -825,6 +825,27 @@ ParameterHandler::leave_subsection()
 
 
 
+bool
+ParameterHandler::subsection_path_exists(
+  const std::vector<std::string> &sub_path) const
+{
+  // Get full path to sub_path (i.e. prepend subsection_path to sub_path).
+  std::vector<std::string> full_path(subsection_path);
```

Comment:
This sounds like the path ``sub_path`` is relative to the currently selected path. Can you make that obvious in the documentation in the .h please?